pages/_document.tsx

```diff
@@ -1,6 +1,8 @@
 import Document, { Html, Head, Main, NextScript, OriginProps } from "next/document";
 import React from "react";
 import { googleTagManager, cspHashOf } from "@lib/cspScripts";
+import JSDisabled from "./js-disabled";
+const noJsScss = `#__next {display:none;}`;
```

Comment:
This is only used on line 222 so I wonder if we should move it there instead of creating a `const` and then using it later on.